pkg/network/ovn_kubernetes.go

```diff
@@ -110,16 +125,27 @@ func fillOVNKubernetesDefaults(conf, previous *operv1.NetworkSpec, hostMTU int)
 	}
 
 	sc := conf.DefaultNetwork.OVNKubernetesConfig
-	// MTU is currently the only field we pull from previous.
-	// If it's not supplied, we infer it from  the node on which we're running.
+	// MTU  and GenevePort are currently the only fields we pull from previous.
```

Comment:
Don't pull GenevePort from previous; we only do it with MTU because if MTU isn't set in previous then we autodetect it. For GenevePort, just set the default value if it's not set in `conf`; see `fillOpenShiftSDNDefaults`.

Reply:
We can't change the port. If its in prev we use it. Where is fillOpenShiftSDNDefaults.

Reply:
OK. Where is fillOpenShiftSDNDefaults?

Reply:
`pkg/network/openshift_sdn.go`

Reply:
Changed

Reply:
Made the change